ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/mapper/defendant/DefendantMapperTest.java

```diff
@@ -357,7 +362,8 @@ public void mapFromCCDDefendantWithSettlements() {
         // Then
         assertNotNull(finalClaim.getSettlementReachedAt());
         assertNotNull(finalClaim.getSettlement());
-        assertThat(finalClaim.getSettlement().get().getPartyStatements().size(), is(3));
+        assertThat(finalClaim.getSettlement().map(Settlement::getPartyStatements).map(List::size).orElse(0),
```

Comment:
This feels like an overcomplicated way of doing this.

Reply:
Maybe : 
```suggestion
assertThat(finalClaim.getSettlement().map( s -> s.getPartyStatements().size()).orElse(0), is(3))
```
Thoughts?

Reply:
Will revert and add a suppression